### DISCUSSION TITLE
How to isolate a category ?

### DISCUSSION THREAD

**Discussion by Sacapofpc** [mapsforge/mapsforge#1431] [Q&A] [answered]
Hello,

How do I isolate the takeoff ? I have this line :
`<rule cat="areas" e="any" k="sport" v="free_flying" >`

But I can't figure out how to proceed further...

**Comment by devemux86** [comment, marked as answer]
It is not very clear what do you mean.

If it is about the themes, you can study the [Mapsforge themes](https://github.com/mapsforge/mapsforge/tree/master/mapsforge-themes/src/main/resources/assets/mapsforge) and see how they are written.

Also OpenStreetMap tags must be present in the map files for themes to render them.
